data/eos-metrics-instrumentation.path.in

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Endless OS Metrics Instrumentation trigger
+Wants=geoclue.service
+After=eos-metrics-event-recorder.service
```

Comment:
Same as above, I don't think this necessarily needs to be here as long as the service has it.